cmd/workflow.go

```
@@ -98,78 +69,27 @@
-						result = append(result, line)
-					}
-				} else if strings.HasPrefix(trimmed, "$") {
-					result = append(result, " "+strings.TrimSpace(line))
-				} else if trimmed != "" {
-					result = append(result, line)
-				}
-			}
-
-			fmt.Print("\n" + strings.Join(result, "\n") + "\n\n")
-			os.Exit(1)
+			cmd.Usage()
```

golangci-lint (1.62.2)

Error return value of `cmd.Usage` is not checked

(errcheck)